### ISSUE TITLE
2016.3.4 Windows minion broke s3 source for file.managed

### ISSUE THREAD

**Issue by wes-novack** [saltstack/salt#38327]
### Description of Issue/Question
Using an S3 source for file.managed state module appears to have been broken in Windows minion version 2016.3.4. This works fine on Windows minion 2016.3.3. Host is running Windows Server 2012 R2.

`Exception: global name '__utils__' is not defined`

### Setup
`yourstate.sls`
```
your_s3_download:
  file.managed:
    - name: c:\yourdir\yourfile.zip
    - source: s3://yourbucket/yourfile.zip
    - skip_verify: True
    - makedirs: True
```

### Steps to Reproduce Issue
```
salt target state.apply yourstate

----------
          ID: your_s3_download
    Function: file.managed
        Name: c:\yourdir\yourfile.zip
      Result: False
     Comment: Failed to cache s3://yourbucket/yourfile.zip: Could not fetch from s3://yourbucket/yourfile.zip. Exception: global name '__utils__' is not defined
     Started: 22:51:16.706000
    Duration: 0.0 ms
     Changes:
```

### Versions Report
Both master and minion are running 2016.3.4

Minion versions report:
```
    Salt Version:
               Salt: 2016.3.4

    Dependency Versions:
               cffi: 1.6.0
           cherrypy: 5.3.0
           dateutil: 2.5.3
              gitdb: 0.6.4
          gitpython: 2.0.2
              ioflo: 1.5.3
             Jinja2: 2.8
            libgit2: Not Installed
            libnacl: 1.4.5
           M2Crypto: Not Installed
               Mako: 1.0.4
       msgpack-pure: Not Installed
     msgpack-python: 0.4.7
       mysql-python: Not Installed
          pycparser: 2.14
           pycrypto: 2.6.1
             pygit2: Not Installed
             Python: 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500 64 bit (AMD64)]
       python-gnupg: 0.3.8
             PyYAML: 3.11
              PyZMQ: 15.2.0
               RAET: Not Installed
              smmap: 0.9.0
            timelib: 0.2.4
            Tornado: 4.3
                ZMQ: 4.1.2

    System Versions:
               dist:
            machine: AMD64
            release: 8.1
             system: Windows
            version: 8.1 6.3.9600  Multiprocessor Free
```

Debug output doesn't seem to give any more details:
```
[INFO    ] Running state [c:\yourdir\yourfile.zip] at time 23:47:48.349000
[INFO    ] Executing state file.managed for C:\yourdir\yourfile.zip
[DEBUG   ] LazyLoaded s3.query
[ERROR   ] Failed to cache s3://yourbucket/yourfile.zip: Could not fetch from s3://yourbucket/yourfile.zip. Exception: global name '__utils__' is not defined
[INFO    ] Completed state [c:\yourdir\yourfile.zip] at time 23:47:48.365000 duration_in_ms=16.0
```

**Comment by gtmanfred** [comment]
I believe that this is a duplicate of https://github.com/saltstack/salt/issues/37388 and will be fixed in 2016.3.5 release.

Can you confirm this and close this issue?

Thanks!
Daniel

**Comment by wes-novack** [comment]
Confirmed. Duplicate of https://github.com/saltstack/salt/issues/37388

Closing.
